crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -69,8 +69,7 @@ const.ERR_ACCOUNT_READING_MAP_VALUE_FROM_NON_MAP_SLOT="failed to read an account
 # The name of the account storage slot at which faucet data is stored.
 # Fungible faucet: The faucet data consists of [0, 0, 0, total_issuance]
 # Non-fungible faucet: The faucet data consists of SMT root containing minted non-fungible assets.
-# TODO(named_slots): Rename constant and procedure to FAUCET_METADATA_SLOT.
-const.FAUCET_STORAGE_DATA_SLOT=word("miden::faucet::metadata")
+const.FAUCET_METADATA_SLOT=word("miden::faucet::metadata")
```

Comment:
I'm still not sure "metadata" is the right term here. For example, for non-fungible faucets we store a set of all issued assets, which isn't really metadata. It is also a bit confusing the the metadata slot in the standard faucet.

My preference (though not a strong one), would be to name this `miden::faucet::sysdata`.